.github/workflows/document_ruby_version.yml

```diff
@@ -24,11 +24,12 @@ jobs:
     with:
       custom_update_command: |
         set -euo pipefail
+        DATE_TODAY="$(date --utc --iso-8601)"
 
-        sed --in-place '/## \[Unreleased\]/a\
-        \
+        sed --in-place "/## \[v${NEW_VERSION}\] - ${DATE_TODAY}/a\\
+        \\
         - ${{ inputs.is_jruby && 'J' || ''}}Ruby ${{inputs.ruby_version}} is now available
-        ' CHANGELOG.md
+        " CHANGELOG.md
```

Comment:
In the generated PR I see an extra trailing newline is added. I think this could be resolved by either moving this closing quote to the line before, or else adjusting the matching term to include a trailing `\n` (perhaps like `${DATE_TODAY}\n` ? though I haven't tested)

Reply:
I added a commit that moves the filename to the same line that we're adding and it has the same outcome:

<img width="621" alt="image" src="https://github.com/user-attachments/assets/76bc442f-699f-42ef-a4dd-bef28d20fd71" />

I think that these lines might come from the prepare release task and it's not considering the use case of an empty prepare. This is what I get using the regular prepare task without adding a version

<img width="1004" alt="image" src="https://github.com/user-attachments/assets/9496ce80-52cf-4376-9bac-8d1a9862df60" />